src/redux/restReducers/meldinger/merkKontorsperret.ts

```diff
@@ -0,0 +1,13 @@
+import { AppState } from '../../reducers';
+import { apiBaseUri } from '../../../api/config';
+import createPostResourceReducerAndActions from '../../../rest/utils/postResource';
+import { KontorsperrRequest } from '../../../models/meldinger/merk';
+
+export function getMerkKontorsperretBaseUrl(state: AppState): string {
+    return `${apiBaseUri}/dialogmerking/kontorsperret`;
+}
+
+export default createPostResourceReducerAndActions<KontorsperrRequest>(
```

Comment:
Trenger vi fult redux-oppsett for å gjøre ett post-kall?

Reply:
Sikkert ikke, men jeg kjenner ikke noen annen måte :-S Forslag?

Reply:
Om vi egentlig bare bryr oss om det går ok eller ikke, så kan man bare bruke `useEffect(() =>{doPostHere(); })`